internal-packages/hub-linker/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@quri/hub-linker",
```

Comment:
Linker package to be reused on website. (It should also be reused in `internal-packages/ai`, but I haven't implemented that yet; will do later).

I think there's not much left to do to implement new schemas (#3495), btw. Having a canonical linker means that we now can add more schemas to it pretty easily.